### ISSUE TITLE
Lots of desync in the mod

### ISSUE THREAD

**Issue by Sveti404** [DJ1TJOO/shapez.io-mods#2]
We have a very big problem with desync. So basically the host and client see things in the map differently and if the host doesn't see it then the progress won't go up.

Here's a few screenshots (client was just building and host visited):

### **Host**
![image](https://user-images.githubusercontent.com/63201551/159356817-d39d29aa-31ec-4963-91b4-083755db1c6b.png)

### **Client**
![image](https://user-images.githubusercontent.com/63201551/159356843-ca1e3d45-03d1-469b-9d3c-1945e9635286.png)

**Comment by VarChar42** [comment]
Yeah, in the current release there is no system to ensure packets are received. From what I have seen in recent commits @DJ1TJOO is working on it.

**Comment by VarChar42** [comment]
![image](https://user-images.githubusercontent.com/46874921/159358582-7cadba3b-730f-4abc-b9b2-e8efe121b51d.png)
Idk, if this is on purpose but in the description the version v0.2.5 is already mentioned, but the latest available release is still 0.2.4

**Comment by DJ1TJOO** [comment]
> ![image](https://user-images.githubusercontent.com/46874921/159358582-7cadba3b-730f-4abc-b9b2-e8efe121b51d.png)
> 
> Idk, if this is on purpose but in the description the version v0.2.5 is already mentioned, but the latest available release is still 0.2.4

The 0.2.5 had some package syncing but it caused too much lag.

But atm I don't have time to work on multiplayer

**Comment by RyanWBell** [comment]
Any update on this issue?

**Comment by DJ1TJOO** [comment]
I'm going to remake multiplayer with a better system, I hope to have some time soon
